### PR TITLE
Add --authfile to podman search

### PR DESCRIPTION
**Pull request by umohnani8**
Since podman search requires credentials to search private registries,
add the --authfile flag to allow users to pass in credentials from a
different authfile than the default one.

Vendor in latest containers/image
Fixes issue with podman search of private registries. Podman search
was not picking up the credentials from the authfile. This fixes it.

Signed-off-by: umohnani8 <umohnani@redhat.com>

### CODE REVIEW

**Comment by umohnani8** [comment]
Fixes https://github.com/projectatomic/libpod/issues/976
@rhatdan @mheon PTAL

**Comment by mheon** [comment]
LGTM

File: vendor.conf [hunk 1]
```diff
@@ -10,7 +10,7 @@ github.com/containerd/cgroups 77e628511d924b13a77cebdc73b757a47f6d751b
 github.com/containerd/continuity master
 github.com/containernetworking/cni v0.6.0
 github.com/containernetworking/plugins 1fb94a4222eafc6f948eacdca9c9f2158b427e53
-github.com/containers/image ad33f7b73fbac0acf05b9e2cea021b61b4b0c3e0
+github.com/containers/image 5144ced37a1b21b63c6ef605e56811e29a687528
```
Comment (TomSweeneyRedHat) [inline vendor.conf]: Is this the same vendor @haircommander is doing?

Comment (haircommander) [reply vendor.conf]: yes ignore mine

**Comment by rhatdan** [comment]
Can we get a test for this?

**Comment by rhatdan** [comment]
LGTM

**Comment by umohnani8** [comment]
Can we? Not sure if we ever got authentication with private registries working for tests. @TomSweeneyRedHat I remember you were working on this?

**Comment by rhatdan** [comment]
Ok lets merge and then we can work on a test later or at least add it to the test script.

**Comment by rhatdan** [comment]
@rh-atomic-bot r+

**Comment by TomSweeneyRedHat** [comment]
Certs are still a sticking point in Travis CI.  Will get it put into the scripts though for at least short term "by hand" testing.

**Pull request closed without merging**